docs/CHANGES.md

```diff
@@ -13,6 +13,16 @@ Changelog
 
 <!-- TOWNCRIER -->
 
+3.6.5 (2020-11-11)
+==================
+
+
+No significant changes.
```

Comment:
I think that the new standard message is `This new version of pulp_installer only differs in that it installs the new version of pulpcore.`
In case you update this PR for other things, consider updating the message as well.
Just the branch is old and the template has the old message.